src/main/java/org/jabref/gui/Base.css

```diff
@@ -1323,10 +1327,6 @@ We want to have a look that matches our icons in the tool-bar */
     -fx-label-padding: 5 0 10 10;
 }
 
-TextFlow * {
-    -fx-fill: -fx-text-background-color;
-}
-
```

Comment:
I'm not sure about the implications of this change. However, Leaving the style as it's will set the hyperlink color to black *(because in the implementation of HyperLinkLabel, the hyperlink is wrapped inside a TextFlow)*

Reply:
I think one of the dialogs related to the OpenOffice integration  which shows citations with context is using TextFlow as well. But would need to check that.

Reply:
I think in general it is a bad idea to change a basic control Style, if you just want it to take effect in one special case. Maybe you make a style class of it instead... Like hyperlinklabel or so.

Reply:
@calixtus you have a point. For instance, if I want to mix icons or shapes with text, it seems like a good default to assign the shapes and icons the same color as the text.